mailpoet/tasks/fix-php82-robo.php

```diff
@@ -0,0 +1,44 @@
+<?php // phpcs:ignore SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing
```

Comment:
In my opinion, we shouldn't add more Code Sniffer ignores when it's not necessary. Do you think we can avoid this ignore?

Reply:
Sure, I've removed it and declared strict types for this file.